inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -333,6 +334,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     pass_config->enable<ngraph::pass::ConvertInterpolate1ToInterpolate4>();
     pass_config->enable<ngraph::pass::ConvertGather1ToGather7>();
     pass_config->enable<ngraph::pass::ConvertGather8ToGather7>();
+    pass_config->enable<ngraph::pass::TransposeReshapeEliminationForMatmul>();
```

Comment:
why do we activate the transformation in the plugin as well? I think the common optimization pass is sufficient.

Reply:
Enabled for all plugins as default. Thanks!